Resources/views/themes/admin/dashboard/block/ez.html.twig

```diff
@@ -0,0 +1,100 @@
+        {# Want to edit these messages? If you are on GPL like any other modications make sure to share your modifcations(/exensions).
+           But to make sure everyone benefits please consider contributing it to ezsystems/ez-support-tools  #}
+        {% if ez.isEndOfMaintenance %}
+            {# In the future with retrival of info from updates.ez.no we can detect missing security fixes and make error for that when public #}
+            {% set badge = '<span class="badge badge-pill badge-warning">GPL</span>' %}
+            <div class="alert alert-warning" role="alert">
+                ⚠️ Unfortunately {{ ez.release }} open source version has reached end of life, <a target="_blank" href="https://doc.ezplatform.com/en/latest/releases/updating_ez_platform/">please upgrade</a>.
+
+                {# Display some options on what might give value when dealing with upgrades #}
+                <em>
+                {% set rand = random(3) %}
+                {% if rand == 0 %}
+                    If you are in need of assistance, eZ Services like <a target="_blank" href="https://ez.no/Services/Consulting">Consulting</a>
+                    and <a target="_blank" href="https://ez.no/Services/Training">Training</a> helps you get the most out your install.
+                {% elseif rand == 1 %}
+                    To simplify the work involved in upgrading, <a target="_blank" href="https://ez.no/Products/eZ-Platform-Cloud">eZ Platform Cloud</a> might be a good option.
+                {% elseif rand == 2 %}
+                    Wondering about adding commerce capabilities while upgrading? <a target="_blank" href="https://ez.no/Products/eZ-Commerce">eZ Commerce</a> could be what you need.
+                {% else %}
+                    If you upgrade to eZ Platform Enterprise you'll get access to:
+                    <a target="_blank" href="https://ez.no/About-our-Software/Licenses-and-agreements">A business friendly license</a>,
+                    <a target="_blank" href="https://ez.no/Products/eZ-Platform-Enterprise-Edition">several productivity features</a>,
+                    <a target="_blank" href="https://ez.no/Services/Support-Maintenance">professional Support</a> and
+                    <a target="_blank" href="https://support.ez.no/Public/Service-Life">longer maintenance of your release</a>.
+                {% endif %}
+                </em>
+            </div>
+        {% else  %}
+            {% set badge = '<span class="badge badge-pill badge-light">GPL</span>' %}
+            <div class="alert alert-info" role="alert">
+                🚀 Welcome to the open source {{ ez.release }} release. Using the community friendly
+                <a target="_blank" href="https://ez.no/About-our-Software/Licenses-and-agreements">GPL</a> license,
+                <a target="_blank" href="https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.en.html#GPLModuleLicense">sharing code</a>
+                is what it's ❤️ about.
```

Comment:
I don't really speak fluent emoji, but is it really supposed to say "what it's heart about"?

Reply:
all